Source/x11/XGServerWindow.m

```diff
@@ -4483,8 +4485,13 @@ - (NSArray *)screenList
               monitors[mi].screen_id = defScreen;
               monitors[mi].depth = [self windowDepthForScreen: mi];
               monitors[mi].resolution = [self resolutionForScreen: defScreen];
-              monitors[mi].frame = NSMakeRect(crtc_info->x, crtc_info->y,
-                                              crtc_info->width, crtc_info->height);
+              /* Transform coordinates from Xlib (flipped) to OpenStep (unflippped). 
+                 Windows and screens should have the same coordinate system. */
+              frame = NSMakeRect(crtc_info->x,
```

Comment:
You don't need this intermediate variable.

Reply:
I use it to make code width less than 80 characters. Anyway I've made changes.